app/assets/src/components/AlignmentViz.jsx

```diff
@@ -1,6 +1,9 @@
-import axios from "axios";
 import React from "react";
+import cx from "classnames";
+import { Popup } from "semantic-ui-react";
+import { getSampleMetadata, fetchAlignmentData } from "~/api";
```

Comment:
nit: can we may keep the prefixes consistent: always `get` ?

Reply:
Definitely, thanks for catching that.